Modules/Loadable/Terminologies/Widgets/qSlicerTerminologySelectorDialog.cxx

```diff
@@ -29,21 +29,17 @@
 #include <QDebug>
 #include <QDialog>
 #include <QHBoxLayout>
-#include <QMessageBox>
 #include <QPushButton>
-#include <QSettings>
 #include <QVBoxLayout>
 
-unsigned int qSlicerTerminologySelectorDialog::CustomNameCounter = 0;
-
 //-----------------------------------------------------------------------------
 class qSlicerTerminologySelectorDialogPrivate : public QDialog
 {
   Q_DECLARE_PUBLIC(qSlicerTerminologySelectorDialog);
 protected:
   qSlicerTerminologySelectorDialog* const q_ptr;
 public:
-  qSlicerTerminologySelectorDialogPrivate(qSlicerTerminologySelectorDialog& object, QWidget* parent);
+  qSlicerTerminologySelectorDialogPrivate(qSlicerTerminologySelectorDialog& object);
```

Comment:
The parent was to improve positioning of the dialog. It's better to appear in the center of the app than the center of the screen, the user needs to move the mouse less.

Reply:
I moved other changes in this file elsewhere and it no longer seemed necessary. I'll add it back. It is important to document this kind of hidden knowledge, to avoid accidental removal.